config/jobs/cert-manager/cert-manager-postsubmits.yaml

```diff
@@ -32,62 +32,6 @@ presets:
 postsubmits:
   jetstack/cert-manager:
 
-  # Publish releases for v0.12.x
-  - name: post-cert-manager-release
```

Comment:
This job already exists as `post-cert-manager-release-previous` in the `release-0.12` directory